TCD-2899: Backtrace Language Integrations (#2148)
saucelabs/sauce-docs: docs/error-reporting/language-integrations/node.md

---
id: node
title: Node Integration Guide
sidebar_label: Node
description: Use Node in your Backtrace project.
---

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';
import useBaseUrl from '@docusaurus/useBaseUrl';

> Backtrace provides an open-source module that you can use to capture uncaught JavaScript errors within Node.js, as well as manually send reports.

Author: ```suggestion
Backtrace provides an open source module that you can use to capture uncaught JavaScript errors in Node.js and manually send reports.
```
